src/server/services/tips-and-notifications.ts

```diff
@@ -30,7 +30,10 @@ import { fetchVergunningenNotifications } from './vergunningen/vergunningen';
 import { fetchWiorNotifications } from './wior';
 import { fetchWpiNotifications } from './wpi';
 
-export function sortNotifications(notifications: MyNotification[]) {
+export function sortNotifications(
+  notifications: MyNotification[],
+  doRandomize: boolean = true
```

Comment:
was this a requirement? I like it, just wondering

Reply:
I've discussed this with Bram and we agreed it would be a nicer way to at least try to differentiate the tips. A better solution would be to reference some persistent state that indicates which tip was shown previously and show a different one next.